### DISCUSSION TITLE
Setting up Outline Google Analytics

### DISCUSSION THREAD

**Discussion by edwarddarrah** [outline/outline#4728] [General] [answered]
I am looking to add Outline to Google Analytics. Has anyone successfully done this? 

Any information on the process or resources would be greatly appreciated.

**Comment by tommoor** [comment, marked as answer]
It's very simple, you go to **Settings -> Integrations -> Google Analytics** and add the Measurement Id from GA. This integration was included in `0.67.1+`

**Reply by edwarddarrah** [reply]
@tommoor Thank you.
